code/modules/cargo/bounties/science.dm

```diff
@@ -64,3 +64,55 @@
 	description = "With the price of rechargers on the rise, upper management is interested in purchasing guns that are self-powered. If you ship one, they'll pay."
 	reward = 10000
 	wanted_types = list(/obj/item/gun/energy/e_gun/nuclear)
+
+/datum/bounty/item/science/bscells
+	name = "Blue Space power cells"
+	description = "Nanotrasen would make good use of those high-capacity 'blue space' cells you have. If you have any, please ship them."
```

Comment:
```suggestion
	description = "Someone in upper management keeps using the excuse that his tablet battery dies when he's in the middle of work. This will be the last time he doesn't have his presentation, I swear to -"
```